tests/Data/Food/RecipeTest.elm

```diff
@@ -211,12 +211,12 @@ suite =
                         ]
                     , describe "getMassAtPackaging"
                         [ { ingredients =
-                                [ { id = Ingredient.idFromString "egg-indoor-code3"
+                                [ { id = Ingredient.idFromString "9cbc31e9-80a4-4b87-ac4b-ddc051c47f69"
                                   , mass = Mass.grams 120
```

Comment:
We need to update `Ingredient.Id` to be now typed as an official `Uuid`: this adds stronger requirements and validation of the format. See how it's done for `Data.Food.Process.Id` (and `Data.Textile.Process.Id` lately)

Reply:
Done!